pegan_text_mining_2.py

```diff
@@ -0,0 +1,81 @@
+'''
+    Alisha Pegan
+    Software Design Spring 2017 Text Mining Assignment
+    Code that produces the top 'n' twitter users that have tweeted a postive
+    sentiment containing a specific hashtag
+    The user inputs the hashtag, and n usernames
+    The code outputs top n twitter usersnames that the person can follow
+    '''
+import os
+import tweepy
+from tweepy import Cursor
+from tweepy import API
+from tweepy import OAuthHandler
+import time
+import pandas
+import pickle
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+# bring in the twitter app keys
+from twitter_knock import Consumer_Key, Consumer_Secret, Access_Token, Access_Secret
+auth = tweepy.OAuthHandler(Consumer_Key, Consumer_Secret)
+auth.set_access_token(Access_Token, Access_Secret)
+
+# set how many tweets to get
+tweet_count = 1000
+# ensure that i do not go over rate limit
+api = tweepy.API(auth, wait_on_rate_limit=True)
+
+
+def pop_tweeters(hashtag_input, list_number):
+    '''
+    function called pop_tweeters to show top tweeters with the
+    most followers who tweet about a certain topic
+    '''
+    # input a hashtag and number
+    hashtag = str(hashtag_input)
+    number = int(list_number)
```

Comment:
Good instincts on checking that the right types are passed in! Generally, you simply do a check like:
``` python
if not isinstance(hashtag_input, str):
    raise TypeError("hashtag_input must be a string!")
```
That code would check if `hashtag_input` is a string, and if not, it would cause an error.